docs/nodes/validator-node.mdx

```diff
@@ -325,7 +350,90 @@ You have the following option of connecting to list of networks shown below:
 Continuing the validator tutorial, here are the steps to connect your
 validator to Mocha:
 
-```sh
+```bash
+MONIKER="your_moniker"
+VALIDATOR_WALLET="validator"
+
+celestia-appd tx staking create-validator \
+    --amount=1000000utia \
+    --pubkey=$(celestia-appd tendermint show-validator) \
+    --moniker=$MONIKER \
+    --chain-id=mocha \
+    --commission-rate=0.1 \
+    --commission-max-rate=0.2 \
+    --commission-max-change-rate=0.01 \
+    --min-self-delegation=1000000 \
+    --from=$VALIDATOR_WALLET \
+    --keyring-backend=test
+```
+
+You will be prompted to confirm the transaction:
+
+```console
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+Inputting `y` should provide an output similar to:
+
+```console
+code: 0
+codespace: ""
+data: ""
+gas_used: "0"
+gas_wanted: "0"
+height: "0"
+info: ""
+logs: []
+raw_log: '[]'
+timestamp: ""
+tx: null
+txhash: <tx-hash>
+```
+
+## Register your validator's EVM address
+
+This section will cover how to register your validator's EVM address.
+This is required if you run an orchestrator. If you don't run an
+orchestrator, you don't need to register your EVM address.
```

Comment:
```suggestion
This is required to run an orchestrator.
```

I guess it's better not to show that validators have a choice not to run the orchestrator